src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -57,12 +62,12 @@ const LanguageNavigation = ({
   setSelectedSDKInterfaceTab,
   setPreviousSDKInterfaceTab,
 }: LanguageNavigationProps) => {
-  const pageLanguage = useContext(PageLanguageContext);
+  const { currentLanguage: pageLanguage, setPreferredLanguage } = usePageLanguage();
```

Comment:
There's a lot of aliasing for this across these changed files, perhaps there's motivation for either adopting `pageLanguage` within the context, or adopting `currentLanguage` in files like this

Reply:
I presume naming convention in project is per component or per feature so that is why, but we could agree on one convention and I can create a separate PR for uniforming that in whole project